Stripe/STPCustomer.m

```diff
@@ -8,15 +8,23 @@
 
 #import "STPCustomer.h"
 
+#import "STPAddress.h"
 #import "STPCard.h"
 #import "STPSource.h"
 #import "StripeError.h"
+#import "NSDictionary+Stripe.h"
 
+/**
+ NOTE: STPCustomer has been deprecated. When we remove STPBackendAPIAdapter,
+ we should rename STPCustomer with STPLimitedCustomer, to avoid confusion –
+ customers retrieved using an ephemeral key have a limited set of fields.
```

Comment:
ah yeah, meant to delete this comment, I think it got carried over from an old version of `sources-ui`. I agree, i think it's fine to stick with STPCustomer.